lib/trailblazer/activity/version.rb

```diff
@@ -1,7 +1,7 @@
 module Trailblazer
   module Version
     module Activity
-      VERSION = "0.9.3"
```

Comment:
no need to bump the version, the pervious one was not released.
It should be 0.9.2